packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -88,6 +89,12 @@ export class DataProxyEngine extends Engine {
     } as GetConfigResult)
   }
 
+  getDmmf(): Promise<DMMF.Document> {
```

Comment:
Maybe add a comment that this path should never be hit as it is handled upstream in `getPrismaClient`.